src/payload-receiver-service/app/main.py

```diff
@@ -0,0 +1,82 @@
+# Standard Library
+import asyncio
+import logging
+import math
+
+# Third Party
+import numpy as np
+import pandas as pd
+from fastapi import FastAPI, HTTPException, Request
+from nats.aio.client import Client as NATS
+from nats_wrapper import NatsWrapper
+
+app = FastAPI()
+logging.basicConfig(level=logging.INFO, format="%(asctime)s - %(name)s - %(message)s")
+nw = None
+
+
+@app.on_event("startup")
+async def startup_event():
+    global nw
```

Comment:
one more minor comments is, nw is already a global variable so we don't need to do `global nw` here.

Reply:
`global nw` needs to be here because `@app.on_event("startup")` is a FastAPI related processes 

It can be removed from `get_nats()` though.